### PR TITLE
#813 Preliminary Input Error Check Enrichment

### PR DESCRIPTION
**Pull request by CaitlinOCallaghan**
Preliminary design that checks gene input and returns alerts.

### CODE REVIEW

File: src/client/features/enrichment/index.js [hunk 1]
```diff
@@ -42,11 +95,22 @@ class Enrichment extends React.Component {
 
   geneInputSubmission(input){
     const geneArray = input.split(/\n/g);
+    if(geneArray.length < 5) return alert("Please input 5 or more tokens");
+    if(geneArray.length > 200)return alert("Please input 200 or less tokens");
     const inputObject = {genes: _.pull(geneArray,"")};
```
Comment (jvwong) [inline src/client/features/enrichment/index.js]: Unnecessary restrictions. Remove.

File: src/client/features/enrichment/index.js [hunk 2]
```diff
@@ -21,6 +21,59 @@ const enrichmentConfig={
   useSearchBar: true
 };
 
+function validateInput(query, array, object)
```
Comment (jvwong) [inline src/client/features/enrichment/index.js]: Add comments here. Take this opportunity to list the task(s) you wish to accomplish here 

Also
- Change function name
- More informative parameter names

**Comment by CaitlinOCallaghan** [comment]
The input box has been updated to provide feedback, but it still needs work. It's a bit choppy because it returns the tokens in a different order than entered, and there are currently some issues when resubmitting. I will look into fixing these problems. 

Also, do we want to differentiate the response if a gene is duplicate vs unrecognized?

**Comment by CaitlinOCallaghan** [comment]
One thing to consider is that if the user puts a space after the gene they typed, ( ex: "tp53 " instead of "tp53" , the validation service will treat the token as unrecognized which can be frustrating for the user

**Comment by jvwong** [comment]
> One thing to consider is that if the user puts a space after the gene they typed, ( ex: "tp53 " instead of "tp53" , the validation service will treat the token as unrecognized which can be frustrating for the user

So this argues for a little helper function that cleans input...

**Comment by maxkfranz** [comment]
We should definitely do a `String.prototype.trim()` on each line.

**Pull request closed without merging**